### PR TITLE
chore: We only use KEDA scaling with visible messages

### PR DESCRIPTION
**Pull request by luisuribe**
## Description

Now we can configure the scaleOnInFlight parameter. Also we don't pull the image of the container by default.

## Checklist

- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: parcellab/common/templates/_keda.tpl [hunk 1]
```diff
@@ -36,6 +36,7 @@ spec:
         queueLength: "{{ $service.keda.queueLength }}"
         queueURL: {{ $service.keda.queueURL }}
         identityOwner: pod
+        scaleOnInFlight: {{ $service.keda.scaleOnInFlight }}
```
Comment (gersmann) [inline parcellab/common/templates/_keda.tpl:39]: should this have a default value?

Comment (andibeuge) [reply parcellab/common/templates/_keda.tpl:39]: yes, totally  - @luisuribe please add a default value when $service.keda.scaleOnInFlight is not set and also add the value to the values.yaml respectively to ensure users of the chart are aware of this setting

Comment (luisuribe) [reply parcellab/common/templates/_keda.tpl]: Done!

File: parcellab/common/templates/_keda.tpl [hunk 2]
```diff
@@ -10,6 +10,7 @@
 {{- define "keda.hpa" -}}
 {{- $service := default dict .service -}}
 {{- $componentValues := (merge (deepCopy .) (dict "component" $service.name)) -}}
+{{- $scaleOnInFlight := (ternary $service.keda.scaleOnInFlight false (eq (typeOf $service.keda.scaleOnInFlight) "bool" )) -}}
```
Comment (gersmann) [inline parcellab/common/templates/_keda.tpl:13]: not sure I get it, this will evaluate to `true` if `scaleOnInFlight` is a bool?

Comment (luisuribe) [reply parcellab/common/templates/_keda.tpl:13]: That's right. If $service.keda.scaleOnInFlight is a bool, then $scaleOnInFlight will have that value. If not, then $scaleOnInFlight will be false.

In other works, any scaleOnInFlight value in the values(-external).yaml file that is not `true` will be changed to `false`

Comment (gersmann) [reply parcellab/common/templates/_keda.tpl:13]: ah, the predicate for ternary is the last parameter, got it.

**Review by gersmann — APPROVED** [review]

**Pull request merged**